src/core/validateGeneratedTypes.ts

```diff
@@ -27,29 +33,64 @@ export function validateGeneratedTypes({
   zodSchemas,
   integrationTests,
   skipParseJSDoc,
+  extraFiles = [],
 }: ValidateGeneratedTypesProps) {
   // Shared configuration
   const compilerOptions: ts.CompilerOptions = {
     target: ts.ScriptTarget.ES2016,
     esModuleInterop: true,
   };
 
-  // Create virtual files system with our 3 files
-  const fsMap = createDefaultMapFromNodeModules({
-    target: compilerOptions.target,
-  });
-  const projectRoot = makePosixPath(process.cwd());
-  const src = fixOptionalAny(
+  const sourceFile = ts.createSourceFile(
+    "index.ts",
```

Comment:
just curious: does this file have to be named "index.ts"? or could it be named anything since this is just the basis for the `fixOptionalAny` ?

Reply:
I think the name doesn't matter but this code was implemented by @fabien0102 and I just moved it for better readability.